### ISSUE TITLE
The type or namespace name `EditorWindow` could not be found

### ISSUE THREAD

**Issue by ColorlessColor** [Dolphiiiin/UVExporter#1]
```
Packages\com.dolphiiiin.uv-exporter\Editor\Scripts\UVExporter.cs(7,31): error CS0246: The type or namespace name 'EditorWindow' could not be found (are you missing a using directive or an assembly reference?)
Packages\com.dolphiiiin.uv-exporter\Editor\Scripts\UVExporter.cs(26,10): error CS0246: The type or namespace name 'MenuItemAttribute' could not be found (are you missing a using directive or an assembly reference?)
Packages\com.dolphiiiin.uv-exporter\Editor\Scripts\UVExporter.cs(26,10): error CS0246: The type or namespace name 'MenuItem' could not be found (are you missing a using directive or an assembly reference?)
```

**Comment by Dolphiiiin** [comment]
Sorry for taking so long to reply.
I have just released a fixed v1.0.2. Please check it out if you still need it.

**Comment by ColorlessColor** [comment]
Thanks for taking the time to fix it!
